test/e2e/cmd_test.go

```diff
@@ -0,0 +1,68 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+)
+
+type cmdParams struct {
+	args []string
+	envs []string
+	dir  string
+}
+
+// runCommand executes a command in a way that resembles an interactive usage
+// more closely. Specifically, it shows stdout/stderr during command execution.
+func runCommand(ctx context.Context, name string, params cmdParams) error {
+	cmd := exec.CommandContext(ctx, name, params.args...)
+	cmd.Env = append(os.Environ(), params.envs...)
+	cmd.Dir = params.dir
+
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		return fmt.Errorf("failed to get stdout pipe: %s", err)
+	}
+	stderr, err := cmd.StderrPipe()
+	if err != nil {
+		return fmt.Errorf("failed to get stderr pipe: %s", err)
+	}
+
+	err = cmd.Start()
+	if err != nil {
+		return fmt.Errorf("failed to start command: %s", err)
+	}
+
+	// Copy stdout/stderr output while the command is running. The io.Copy
+	// operations will terminate once the command finishes.
```

Comment:
Nit: `os/exec` will do this for us if we set `cmd.Stdout` and `cmd.Stderr` to `os.Stdout` and `os.Stderr` respectively (see [comments on Stdout and Stderr in the docs](https://godoc.org/os/exec#Cmd)). Is there a reason to do it ourselves rather than leaning on the standard library?

Reply:
Coincidentally, I only learned about `cmd.Std{out,err}` providing the same functionality as I built a few days ago. I forgot to follow up in my PR though, so glad you brought it up as well.

I adjusted the code accordingly, which allows us to throw out a fair number of lines. 👍